.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on:
+- pull_request
+
+jobs:
+  sanity:
+    name: Sanity (${{ matrix.ansible }})
+    strategy:
+      matrix:
+        ansible:
+        - stable-2.10
+        - devel
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out code
+        uses: actions/checkout@v1
+        with:
+          path: ansible_collections/dellemc/os10
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
+
```

Comment:
Add a new section here which does ansible-galaxy collection install ansible.netcommon -p ../..

Reply:
Thanks @gundalow  for the continuous support. WIth this suggestion, the sanity test ran fine.